### ISSUE TITLE
Not working with newer toolchains

### ISSUE THREAD

**Issue by fabi321** [rustyscreeps/cargo-screeps#29]
Starting with rustc version 1.52.0 till present, cargo_screeps panics with the following message
```
thread 'main' panicked at 'unknown name section chunk type: 7', /home/.cargo/registry/src/github.com/cargo-web-0.6.26/src/wasm_context.rs:642:41
```

**Comment by smessmer** [comment]
@daboross are there any plans to fix this and https://github.com/rustyscreeps/cargo-screeps/issues/30 to make it work for newer rust versions, or is this project dead?

**Comment by daboross** [comment]
I'm not currently working on this project. If you're interested (or if anyone else is) in using this, I could definitely provide pointers or see how hard it would be to get it working on the latest version of rust, though! I think the starting point would be finishing PR - https://github.com/rustyscreeps/cargo-screeps/pull/24, since our dependency on `cargo-web` (which has been deprecated and is no longer updated) is the main thing broken here.

There are also a few other maintainers who I haven't been in contact with who may or may not be still using the project. I haven't been in contact with them for quite a while, but assuming everyone I knew switched over when the Screeps Slack community moved over to Discord, the best place to check in and contact them would be #rust in the Screeps Discord (https://discord.com/invite/RjSS5fQuFx).

I'm probably not going to start maintaining it actively, but I could definitely try to fix the build system and review PRs.

**Comment by shanemadden** [comment]
Resolved by #24
